README.md

```diff
@@ -123,6 +134,10 @@ Here is a list and description of the different arguments to use for the script:
 * If specified, will use the specified driver instead of the defaults
 * NOTE: The driver should match the browser specified in the `--driver-type` option (defaults to Firefox)
 
+<b>--cvv</b>
```

Comment:
This is already described in the README.

Reply:
Got it 👍